### ISSUE TITLE
Couldn't make this work

### ISSUE THREAD

**Issue by JJ** [ripcurld00d/logrus-logzio-hook#4]
it fails rather silently. The problem seems to be somewhere in the definition of host, but I have tried listener.logz.io, with or without https, with or without port number. Any help (or example host configuration) here will be appreciated.

**Comment by ripcurld00d** [comment]
Well, I assumed the logzio instance is using the [http plugin](https://www.elastic.co/blog/introducing-logstash-input-http-plugin) and tested it on that case.

Can you provide more details. For example, are you sure the logzio instance is up?
Did you try to `curl` that endpoint?

Thanks.

**Comment by JJ** [comment]
I'll try that, but meanwhile, what would be the correct URL or host:port
combination?

**Comment by ripcurld00d** [comment]
```go
hook := logzio.New("http://listener.logz.io:9292", "CuantoQuedaBot", fields)
```
where `9292` is the port number, `listener.logz.io` is the hostname and `http` is the protocol.

**Comment by JJ** [comment]
Thanks!

**Comment by ripcurld00d** [comment]
No problem. Let me know if you need anything.

**Comment by JJ** [comment]
I'm afraid I'm useless with this. I have tested it with the above configuration, nothing goes. I've tried to use it with bulk http, and the example itself does not work. I don't know if I'm missing something, maybe some NAT magic on my home router, something like that. It just swallows the request, does not do anything. I'm closing it since it seems to be my fault anyway. It does not work on the cloud either, so there's that too. 
 Thanks for your help.
